### PR TITLE
Contributing update, referencing #152

### PR DESCRIPTION
**Pull request by divayprakash**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,10 +8,25 @@
 0. Clone your fork of the repo: `git clone git@github.com:<YOUR-USERNAME>/zen-audio-player.github.io.git`.
 0. Navigate to the project: `cd zen-audio-player.github.io`.
 0. Install test dependencies: `npm install`.
+0. If you've started working on an issue, please ensure that you leave a comment as it helps avoid duplicate work.
 0. Make your changes.
 0. Run the tests: `npm test`.
 0. Run `git commit` and add your message describing what you have changed. If you have made changes that address an issue include the text `Closes #1` (where 1 would be the issue number) to your commit message.
 0. Submit a [pull request](https://github.com/zen-audio-player/zen-audio-player.github.io/pulls) - Travis CI will run tests to make sure nothing broke.
+0. In case the following occurs, skip to **Syncing** below
```
Comment (shakeelmohamed) [inline .github/CONTRIBUTING.md]: We can add a link to `**Syncing**`, I think `**[Syncing](#syncing)**` will work

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -8,10 +8,25 @@
...
+![out of sync](/.github/outofsync.png)
+0. After you've had 1 PR merged, [@shakeelmohamed](http://github.com/shakeelmohamed) will add you to the organization. Then you can create any issues you work on before you start.
+
+### Syncing  
+
+This issue arises when the `master` branch of the upstream repository has been changed (ie. new commits added) since you checked out a development branch. It can be resolved in the following manner -  
+  
+0. Click on the **command line instructions**  as shown in the image above. The instructions are also explained below.  
+0. Navigate to the current working directory of your working project on the command line.  
+0. Use `git fetch upstream` to fetch the branches and their respective commits from the upstream repository and store them locally. Commits to the upstream `master` branch are stored in a local branch `upstream/master`.
```
Comment (shakeelmohamed) [inline .github/CONTRIBUTING.md]: `upstream` is not necessarily the the remote name. In my few years of git experience, it's usually `origin`. I'd add another step here saying something like:

* Run `git remote` to determine the remote name, if you have multiple remote configured, you can run `git remote -v` to also see the URLs associated with each remote.

**Comment by shakeelmohamed** [comment]
While we're at it, can we move the contact section to the top?

**Comment by divayprakash** [comment]
Above the **How to contribute** or just above **Getting Started**?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -8,10 +13,26 @@
 0. Clone your fork of the repo: `git clone git@github.com:<YOUR-USERNAME>/zen-audio-player.github.io.git`.
 0. Navigate to the project: `cd zen-audio-player.github.io`.
 0. Install test dependencies: `npm install`.
+0. If you've started working on an issue, please ensure that you leave a comment as it helps avoid duplicate work.
```
Comment (shakeelmohamed) [inline .github/CONTRIBUTING.md]: Remove `please ensure that you`

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -8,10 +13,26 @@
...
 0. Run the tests: `npm test`.
 0. Run `git commit` and add your message describing what you have changed. If you have made changes that address an issue include the text `Closes #1` (where 1 would be the issue number) to your commit message.
 0. Submit a [pull request](https://github.com/zen-audio-player/zen-audio-player.github.io/pulls) - Travis CI will run tests to make sure nothing broke.
+0. In case the following occurs, skip to **[Syncing](#syncing)** below  
+![out of sync](/.github/outofsync.png)
+0. After you've had 1 PR merged, [@shakeelmohamed](http://github.com/shakeelmohamed) will add you to the organization. Then you can create any issues you work on before you start.
+
+### Syncing  
+
+This issue arises when the `master` branch of the upstream repository has been changed (ie. new commits added) since you checked out a development branch. It can be resolved in the following manner -
```
Comment (shakeelmohamed) [inline .github/CONTRIBUTING.md]: nit: should be `i.e`

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -8,10 +13,26 @@
...
+![out of sync](/.github/outofsync.png)
+0. After you've had 1 PR merged, [@shakeelmohamed](http://github.com/shakeelmohamed) will add you to the organization. Then you can create any issues you work on before you start.
+
+### Syncing  
+
+This issue arises when the `master` branch of the upstream repository has been changed (ie. new commits added) since you checked out a development branch. It can be resolved in the following manner -  
+  
+0. Click on the **command line instructions**  as shown in the image above. The instructions are also explained below.  
+0. Navigate to the current working directory of your working project on the command line. 
+0. Run `git remote` to determine the remote name, if you have multiple remotes configured, you can run `git remote` -v to also see the URLs associated with each remote.
```
Comment (shakeelmohamed) [inline .github/CONTRIBUTING.md]: Adding to this point:

```markdown
The remote you want will have a URL containing `github.com/zen-audio-player/zen-audio-player.github.io`.
```

**Comment by shakeelmohamed** [comment]
Thanks!

**Pull request merged**